### PR TITLE
documented process for updating maintainers

### PR DESCRIPTION
**Pull request by zr-msft**

### CODE REVIEW

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
cc @iamsamirzon @yizha1 @priteshbandi @toddysm for reviewing this PR

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,3 +48,9 @@ $ npm run check-links
     git push origin HEAD
     ```
 1. File a PR to [notaryproject.dev](https://github.com/notaryproject/notaryproject.dev).
+
+## Updating project maintainers
+
+1. Create a PR that updates both [CODEOWNERS]() and [MAINTAINERS]() with the maintainer updates. [Example](https://github.com/notaryproject/notaryproject.dev/pull/310/files)
+1. Ensure justification is provided for the change. [Example](https://github.com/notaryproject/.github/issues/34#issue-1753758548)
```
Comment (yizha1) [inline CONTRIBUTING.md:55]: I think this step should come first before creating a PR to update maintainers

**Review by yizha1 — COMMENTED** [review]
Thanks @zr-msft

File: CONTRIBUTING.md (hunk 1 above)
Comment (zr-msft) [reply CONTRIBUTING.md:54]: fixed

**Review by yizha1 — APPROVED** [review]
Thanks @zr-msft 

LGTM

**Comment by zr-msft** [comment]
@FeynmanZhou  and @toddysm mind taking a look?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**